Scripts/NuGet/EdFi.RestApi.Databases/PostDeploy.ps1

```diff
@@ -52,7 +52,5 @@ Add-Parameter $deploymentParams 'ExcludedExtensionSources' $ExcludedExtensionSou
 Add-Parameter $deploymentParams 'EnabledFeatureNames' $EnabledFeatureNames
 if ([Boolean]::TryParse((Add-Parameter $deploymentParams 'UsePlugins' $UsePlugins).UsePlugins, [ref]$UsePlugins)) { $deploymentParams.UsePlugins = $UsePlugins }
 
-If (-Not(Get-IsWindows)) {Add-Parameter $deploymentParams 'Engine' "PostgreSQL"} 
```

Comment:
This is breaking Octopus deployments since `cross-platform.psm1` is not imported, so `Get-IsWindows` isn't defined.
I'm reverting this change since we only use SQL Server in our Octopus deployments; consider adding an `Engine` parameter instead if Postgres has to be supported.